src/models/vae.jl

```diff
@@ -16,9 +16,13 @@ safe_softplus(x::T) where T  = softplus(x) + T(0.000001)
 
 
 """
-	vae_constructor(;idim::Int=1, zdim::Int=1, activation = "relu", hdim=128, nlayers=3, kwargs...)
+	vae_constructor(;;idim::Int=1, zdim::Int=1, activation = "relu", hdim=128, nlayers::Int=3, 
+		init_seed=nothing, prior="normal", pseudoinput_mean=nothing, k=1, kwargs...)
+
+Constructs a classical variational autoencoder.
```

Comment:
Add comment about the additional parameters as it is no longer supporting only classical VAE.

Reply:
Good idea.